### DISCUSSION TITLE
Iframes without transparent-Funktion

### DISCUSSION THREAD

**Discussion by Dennis-89** [grafana/grafana#38165] [Dashboards, Panels & Visualizations]
**What happened**:
If I have created a label and use the 'share' option to embed it as an 'iframe' in a web page, the 'transparent' function does not work. That means in the website I always have a light or dark background of the label. This does not look nice if the website has its own background image.

![grafik](https://user-images.githubusercontent.com/63751687/97104793-de0a2d80-16b6-11eb-99c4-a5ccd5bd969b.png)

**What you expected to happen**:
I expected that the 'transparent' function would also work for iframes.

**How to reproduce it (as minimally and precisely as possible)**:
Create a label. Select the option 'transparent'. Select 'share' to embed the label as iframe in a web page.
Create a simple web page with a colored background and the created label.

**Anything else we need to know?**:

**Environment**:
- Grafana version: 7.2.1
- Data source type & version: MySql
- OS Grafana is installed on: Raspberry Pi OS (32-Bit)
- User OS & Browser: Fedora 32 with Firefox 81.0.1
- Grafana plugins:MySql
- Others:

Thank you.

With kind regards
Dennis

**Comment by torkelo** [comment]
A transparent panel background just means it has the same background as the dashboard background. 

Grafana does currently not support transparent dashboards / panels in iframes

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by Standarduser** [comment]
To get transparent background on embedded panels:

- on your grafana machine go to `/usr/share/grafana/public/views/`
- make backup of `index.html`
- edit `index.html`
- add to first `style`-tag:
  - `:root { color-scheme: none !important; }`
  - `body.dashboard-solo { background: none; }`
- save changes and exit
- restart grafana `systemctl restart grafana-server`

Now embedded (solo viewed) panels have transparent background but grafana dashboard remains as it is

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
